### PR TITLE
Release libkiwix 13.0.0

### PR DESCRIPTION
**Pull request by kelson42**

### CODE REVIEW

File: kiwixbuild/versions.py [hunk 1]
```diff
@@ -31,9 +31,9 @@
 
 release_versions = {
     'libzim': None, # Depends of base deps (was None)
-    'libkiwix': None, # Depends of libzim (was 0)
+    'libkiwix': 0, # Depends of libzim (was 0)
     'kiwix-tools': None, # Depends of libkiwix and libzim (was 2)
-    'zim-tools': 0, # Depends of libzim (was None)
+    'zim-tools': None, # Depends of libzim (was None)
     'kiwix-desktop': None # Depends of libkiwix and libzim (was 4)
```
Comment (mgautierfr) [inline kiwixbuild/versions.py:37]: We should probably also build kiwix-tools and kiwix-desktop to have new binaries using the new libkiwix.

**Review by mgautierfr — CHANGES_REQUESTED** [review]

File: kiwixbuild/versions.py (hunk 1 above)
Comment (kelson42) [reply kiwixbuild/versions.py:37]: New build is probably not possible.
New release could be done for kiwix-tools, but not urgency. I will do it later.

Comment (mgautierfr) [reply kiwixbuild/versions.py:37]: Why not possible ?

Comment (mgautierfr) [reply kiwixbuild/versions.py:37]: Indeed, this is a major version.

**Pull request merged**